src/plugins/tree-shaking.ts

```diff
@@ -25,7 +24,6 @@ export const treeShakingPlugin = ({
       const bundle = await rollup({
         input: [info.path],
         plugins: [
-          hashbang(),
```

Comment:
Shebang is preserved by rollup now.
https://rollupjs.org/migration/#:~:text=When%20bundling%20CLI%20apps%2C%20Rollup%20will%20now%20automatically%20preserve%20shebang%20comments%20in%20entry%20files%20if%20the%20output%20format%20is%20es%20or%20cjs.%20Previously%2C%20you%20would%20have%20needed%20to%20add%20the%20comment%20via%20a%20plugin.